### PR TITLE
ETL setup

### PR DESCRIPTION
**Pull request by davidguy3237**
I think I've managed to fix my photos csv. Although it's hard to tell when it's 5 millions rows long.

### CODE REVIEW

**Comment by davidguy3237** [comment]
[ETL CSV files into postgreSQL](https://trello.com/c/oiTTHkGH/8-etl-csv-files-into-postgresql)

**Review by noahbeito — APPROVED** [review]
Looking good

File: server/db.js
Comment (noahbeito) [inline server/db.js:1]: Likely want to store the host, user and password in your env file and then import those variables to be used when creating client instance.

Comment (davidguy3237) [reply server/db.js:1]: Yeah I'm planning on doing that at some point, just being lazy

**Review by MiladMoulayi — APPROVED** [review]
LGTM!

**Pull request merged**